### PR TITLE
[To rel/0.13][IOTDB-3969]Fix timeseries statistic bug

### PR DESCRIPTION
**Pull request by MarcosZyk**
### Problem
When delete storage group, the timeseries represent by template will be counted and the result will be used to update the timeseries statistic number. The number doesn't update when activate template. Thus the number will be negative.

### Solution
Use two number to statistic template series and normal series separately.

### CODE REVIEW

**Review by bigreybear — APPROVED** [review]
LGTM~

File: server/src/main/java/org/apache/iotdb/db/metadata/MManager.java [hunk 1]
```diff
@@ -319,10 +321,20 @@ private void startStatisticCounts() {
...
+
+    MetricsService.getInstance()
+        .getMetricManager()
+        .getOrCreateAutoGauge(
+            Metric.QUANTITY.toString(),
+            MetricLevel.IMPORTANT,
+            totalTemplateSeriesNumber,
+            AtomicLong::get,
+            Tag.NAME.toString(),
+            "template-timeSeries");
```
Comment (SpriCoder) [inline server/src/main/java/org/apache/iotdb/db/metadata/MManager.java:337]: From my perspective, I think these two metrics can use `type` tag to distinguish.
For example,  TAG.NAME.toString(), "timeSeries", TAG.TYPE.toString(), "normal/template"
Besides, please update Metric doc at the same time: 
1. docs/UserGuide/Maintenance-Tools/Metric-Tool.md
2. docs/zh/UserGuide/Maintenance-Tools/Metric-Tool.md

**Review by SpriCoder — CHANGES_REQUESTED** [review]
PTAL~

File: server/src/main/java/org/apache/iotdb/db/metadata/MManager.java (hunk 1 above)
Comment (MarcosZyk) [reply server/src/main/java/org/apache/iotdb/db/metadata/MManager.java]: Fixed.

**Review by SpriCoder — APPROVED** [review]
LGTM~

**Pull request merged**